frontend/src/scenes/web-analytics/webAnalyticsLogic.tsx

```diff
@@ -1561,20 +1563,25 @@ export const webAnalyticsLogic = kea<webAnalyticsLogicType>([
         statusCheck: {
             __default: null as WebAnalyticsStatusCheck | null,
             loadStatusCheck: async (): Promise<WebAnalyticsStatusCheck> => {
-                const [pageviewResult, pageleaveResult, pageleaveScroll] = await Promise.allSettled([
-                    api.eventDefinitions.list({
-                        event_type: EventDefinitionType.Event,
-                        search: '$pageview',
-                    }),
-                    api.eventDefinitions.list({
-                        event_type: EventDefinitionType.Event,
-                        search: '$pageleave',
-                    }),
-                    api.propertyDefinitions.list({
-                        event_names: ['$pageleave'],
-                        properties: ['$prev_pageview_max_content_percentage'],
-                    }),
-                ])
+                const [pageviewResult, pageleaveResult, pageleaveScroll, pageViewsWithoutSessionsResult] =
+                    await Promise.allSettled([
+                        api.eventDefinitions.list({
+                            event_type: EventDefinitionType.Event,
+                            search: '$pageview',
+                        }),
+                        api.eventDefinitions.list({
+                            event_type: EventDefinitionType.Event,
+                            search: '$pageleave',
+                        }),
+                        api.propertyDefinitions.list({
+                            event_names: ['$pageleave'],
+                            properties: ['$prev_pageview_max_content_percentage'],
+                        }),
+                        api.query<HogQLQuery>({
+                            kind: NodeKind.HogQLQuery,
+                            query: "SELECT events.properties.$lib as lib FROM events WHERE event='$pageview' AND or(events.$session_id IS NULL, events.$session_id = '') AND timestamp > today() LIMIT 1",
```

Comment:
Maybe we should `AND lib IS NOT NULL`? I know this won't grab some of the offending events, but it might give us more meaningful results in the rare case where there are some events with a lib and some without it.

Reply:
Hmm, I think it comes down to whether it's more important to show any error at all, or to show a more useful docs link. IMO I'd rather make sure we are showing any error at all.

Reply:
I agree with that :)